R/Lab 1.Rmd

```diff
@@ -0,0 +1,197 @@
+---
```

Comment:
I just tried locally and it's not working like I would expect. I think leaving it as hosted on github is OK for now but we should probably figure it out at some point.